docs/getting-started/local-setup/setting-up-a-local-chain.md

```diff
@@ -52,7 +64,214 @@ to delete this directory and start the node up again. If you are using the defau
+  -block-time-iota 100           the block time iota (in ms)
+  -chain-id dev                  the ID of the chain
+  -genesis-time 1715163944       the genesis creation time. Defaults to current time
+  -output-path ./genesis.json    the output path for the genesis.json
+```
+
+### 3. Add the initial validator set
+
+A new Gno chain cannot advance without an active validator set.
+Since this example follows starting a completely new Gno chain, you need to add at least one validator to the validator
+set.
+
+Luckily, we've generated the node secrets in step #1 -- we will utilize the generated node key, so the process we start
+locally will be the validator node for the new Gno network.
+
+To display the generated node key data, run the following command:
+
+```shell
+gnoland secrets get ValidatorPrivateKey
+```
+
+This will display the information we need for updating the `genesis.json`:
+
+```shell
+[Validator Key Info]
+
+Address:     g10e3smsmusjn00n7j75fk9u4zta8djrlglcv6af
+Public Key:  gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zqhjhrqd7xlhda7spfdtx6lrcjxlk67av46w7eng9z4e2ch478fsk4xmq3j
+```
+
+Updating the `genesis.json` is relatively simple, running the following command will add the generated node info to the
+validator set:
+
+```shell
+gnoland genesis validator add \
+--address g10e3smsmusjn00n7j75fk9u4zta8djrlglcv6af \
+--pub-key gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zqhjhrqd7xlhda7spfdtx6lrcjxlk67av46w7eng9z4e2ch478fsk4xmq3j \
+--name Cuttlas
+```
+
+We can verify that the new validator was indeed added to the validator set:
+
+```json
+{
+  "genesis_time": "2024-05-08T10:25:09Z",
+  "chain_id": "dev",
+  "consensus_params": {
+    "Block": {
+      "MaxTxBytes": "1000000",
+      "MaxDataBytes": "2000000",
+      "MaxBlockBytes": "0",
+      "MaxGas": "10000000",
+      "TimeIotaMS": "100"
+    },
+    "Validator": {
+      "PubKeyTypeURLs": [
+        "/tm.PubKeyEd25519"
+      ]
+    }
+  },
+  "validators": [
+    {
+      "address": "g1lz2ez3ceeds9f6jllwy7u0hvkphuuv0plcc8pp",
+      "pub_key": {
+        "@type": "/tm.PubKeyEd25519",
+        "value": "AvaVf/cH84urHNuS1lo3DYmtEErxkTLRsrcr71QoAr4="
+      },
+      "power": "1",
+      "name": "Cuttlas"
+    }
+  ],
+  "app_hash": null
+}
+```
+
+### 4. Starting the chain
+
+We have completed the main aspects of setting up a node:
+
+- generated the node directory (secrets and configuration) ✅
+- generated a `genesis.json` ✅
+- added an initial validator set to the `genesis.json` ✅
+
+Now, we can go ahead and start the Gno chain for the first time, by running:
+
+```shell
+gnoland start \
+--genesis ./genesis.json \
+--data-dir ./gnoland-data
+```
+
+That's it! 🎉
+
+Your new Gno node (chain) should be up and running:
+
+![gnoland start](../../assets/getting-started/local-setup/setting-up-a-local-chain/gnoland-start-manual.gif)
+
+## Chain runtime options
+
+### Changing the chain ID
 
 :::info Changing the Gno chain ID has several implications
```

Comment:
Please keep consistency with other info blocks in the file

Reply:
Can you elaborate?

Reply:
In other info sections, you have a column on the end of the first line, and you are not capitalizing the specific items.

Reply:
I'm sorry, I still don't understand. Are you talking about the list inside this info block?

Reply:
Correct: for example, this block

```
:::info Lazy init

Starting a Gno blockchain node using just the `gnoland start` command implies a few things:

- the default configuration will be used, and generated on disk in the `gnoland-data` directory
- random secrets data will be generated (node private keys, networking keys...)
- an entirely new `genesis.json` will be used, and generated on disk in the `../gnoland-data` directory. The genesis
  will have a single validator, whose public key is derived from the previously generated node secrets
```

- Has a "title",
- `Starting a Gno blockchain node using just the gnoland start command implies a few things:` ends with a column
- The items are not capitalized

In this specific one, its different
```
:::info Changing the Gno chain ID has several implications

- It affects how the Gno node communicates with other Gno nodes / chains
- Gno clients that communicate through JSON-RPC need to match this value
```

I suggest the following change:

```
:::info Changing the chain ID

Below are some implications to consider before changing the chain ID:

- it affects how the Gno node communicates with other Gno nodes / chains
- Gno clients that communicate through JSON-RPC need to match this value
```

Hope it's more clear now.